lib/scim_rails/version.rb

```diff
@@ -1,3 +1,3 @@
 module ScimRails
-  VERSION = '0.1.4'
+  VERSION = "0.2.0"
```

Comment:
I bumped the version so that we can deploy it and use it in our app. I gave it a bump in minor version since it adds functionality in that this update allows usage in more versions of rails.

I opted not to bump the patch version since we're not fixing any functionality.